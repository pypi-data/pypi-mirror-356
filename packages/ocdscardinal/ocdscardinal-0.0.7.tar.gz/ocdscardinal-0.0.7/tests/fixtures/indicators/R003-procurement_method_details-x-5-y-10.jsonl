{"ocid":"x","tender":{"tenderPeriod":{"startDate":"2000-01-01T18:00:00Z","endDate":"2000-01-06T17:59:59Z"},"procurementMethodDetails":"x"}}
{"ocid":"x exact","tender":{"tenderPeriod":{"startDate":"2000-01-01T18:00:00Z","endDate":"2000-01-06T18:00:00Z"},"procurementMethodDetails":"x"}}
{"ocid":"y","tender":{"tenderPeriod":{"startDate":"2000-01-01T18:00:00Z","endDate":"2000-01-11T17:59:59Z"},"procurementMethodDetails":"y"}}
{"ocid":"y exact","tender":{"tenderPeriod":{"startDate":"2000-01-01T18:00:00Z","endDate":"2000-01-11T18:00:00Z"},"procurementMethodDetails":"y"}}
{"ocid":"default","tender":{"tenderPeriod":{"startDate":"2000-01-01T18:00:00Z","endDate":"2000-01-16T17:59:59Z"},"procurementMethodDetails":"z"}}
{"ocid":"default exact","tender":{"tenderPeriod":{"startDate":"2000-01-01T18:00:00Z","endDate":"2000-01-16T18:00:00Z"},"procurementMethodDetails":"z"}}
{"ocid":"missing","tender":{"tenderPeriod":{"startDate":"2000-01-01T18:00:00Z","endDate":"2000-01-16T17:59:59Z"}}}
