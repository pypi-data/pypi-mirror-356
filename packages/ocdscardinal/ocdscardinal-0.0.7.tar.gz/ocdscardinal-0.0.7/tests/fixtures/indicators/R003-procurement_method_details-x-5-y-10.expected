{"OCID":{"x":{"R003":1.0},"y":{"R003":1.0},"default":{"R003":1.0},"missing":{"R003":1.0}}}